.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
remove and add to to `.gitignore`

Reply:
As discussed, remove non-mandatory fields asides from `"eslint.validate": ["typescript"]` to enable formatting on a workspace level when using VSC.